### ISSUE TITLE
Process .ZON file

### ISSUE THREAD

**Issue by serrat839** [SvenPaterson/VR6000#1]
Hello, I was wondering if you were processing/using the ZON files that keyence produces from their VR 6000 in this work or if you were exporting the information in a different way. How did you get your test-scan.dxf file?

Thank you!

**Comment by SvenPaterson** [comment]
Yeah, so unfortunately you have to use the VR analysis software to export a .zon file to CAD format. This also has the unfortunate side effect of ballooning the file size by 3-5x. I wasn't able to get my script to do alignment good enough for our purposes so I benched my efforts on doing it outside of the VR software. Just crossing my fingers that keyence implements rotational scanning stitching functionality.
